### PR TITLE
chore: siwx bitcoin jsdocs comments

### PR DESCRIPTION
**Pull request by zoruka**
# Description

Add jsdocs comments explaining params and functions from SIWX and Bitcoin packages.

## Type of change

- [x] Chore (non-breaking change that addresses non-functional tasks, maintenance, or code quality improvements)
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

# Associated Issues

For Linear issues: Closes APKT-xxx
For GH issues: closes #...

# Showcase (Optional)

If there is a UI change include the screenshots with before and after state.
If new feature is being introduced, include the link to demo recording.

# Checklist

- [ ] Code in this PR is covered by automated tests (Unit tests, E2E tests)
- [ ] My changes generate no new warnings
- [ ] I have reviewed my own code
- [ ] I have filled out all required sections
- [ ] I have tested my changes on the preview link
- [ ] Approver of this PR confirms that the changes are tested on the preview link

### CODE REVIEW

File: packages/adapters/bitcoin/src/utils/BitcoinConnector.ts [hunk 1]
```diff
@@ -10,37 +13,80 @@ export interface BitcoinConnector extends ChainAdapterConnector, Provider {
...
   }
 
   export type SendTransferParams = {
+    /**
+     * The amount to be sent in satoshis
+     */
     amount: string
+    /**
+     * The address to send the transfer to
+     */
```
[the remark this answers is not in the document]
Comment (zoruka) [reply packages/adapters/bitcoin/src/utils/BitcoinConnector.ts:52]: no, i dont think it should

File: packages/siwx/src/core/SIWXMessenger.ts [hunk 2]
```diff
@@ -1,17 +1,42 @@
...
+ */
 export abstract class SIWXMessenger {
   public domain: SIWXMessage['domain']
   public uri: SIWXMessage['uri']
   public statement?: SIWXMessage['statement']
   public resources?: SIWXMessage['resources']
 
+  /**
+   * The version of the SIWXMessage class
+   */
```
Comment (tomiir) [inline packages/siwx/src/core/SIWXMessenger.ts:15]: what's the current / defualt version?

File: packages/siwx/src/core/SIWXMessenger.ts [hunk 3]
```diff
@@ -47,6 +80,13 @@ export abstract class SIWXMessenger {
     return Object.assign(params, methods)
   }
 
+  /**
+   * This method generates the expiration time based on the `notBefore` value and the `expiration` value.
+   * You may override this method to provide a custom expiration time.
+   *
+   * @param Pick<SIWXMessage.Input, 'notBefore'>
+   * @returns string | undefined - The expiration time in `stringifyDate` format or undefined if not available
+   */
```
Comment (tomiir) [inline packages/siwx/src/core/SIWXMessenger.ts:89]: is this a utility? or is this core to the package?

File: packages/siwx/src/core/SIWXMessenger.ts (hunk 2 above)
Comment (zoruka) [reply packages/siwx/src/core/SIWXMessenger.ts:15]: there is no default version, this is an abstract class, the one extending it must provide a version

File: packages/siwx/src/core/SIWXMessenger.ts (hunk 3 above)
Comment (zoruka) [reply packages/siwx/src/core/SIWXMessenger.ts:89]: this is an abstract class that must be extended to provide the functionality, this is only predefined logic

**Pull request merged**